ente-io/photos-web
# Monorepo

## Description

This pull request introduces changes to convert the `photos-web` project into a Turborepo monorepo. The conversion aims to improve the development experience and optimize build times using Turborepo's caching and parallelization features.

Key changes include:

1. Setting up Turborepo: The project now uses Turborepo for managing tasks and dependencies across workspaces.

2. Monorepo structure: The project has been restructured into a monorepo, with separate workspaces for different parts of the application.

3. Configuration updates: The `turbo.json` and `package.json` files have been updated to accommodate the new monorepo structure and to utilize Turborepo's features.

4. TypeScript configuration: The `tsconfig.json` files have been adjusted to work with the new monorepo setup.

## Test Plan

1. Move your `.env` to `apps/photos`.
2. Run `yarn dev` at the root of the monorepo.
3. Confirm the app runs properly.
